pyproject.toml

```diff
@@ -68,9 +66,8 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-    # These are super-old; can/should we update them?
-    "cffi>=1.11",
-    "numpy>=1.19",
+    "cffi>=1.15",
+    "numpy>=1.23",
```

Comment:
Is this compatible with numpy 2?

Reply:
Yes. Wheels built with numpy 2 are compatible with older numpy versions, but the reverse is not true.

Reply:
See: https://numpy.org/doc/stable/dev/depending_on_numpy.html#numpy-2-0-specific-advice